module-services/service-desktop/ServiceDesktop.cpp

```diff
@@ -199,7 +199,7 @@ auto ServiceDesktop::usbWorkerInit() -> sys::ReturnCodes
                                                     std::bind(&ServiceDesktop::restartConnectionActiveTimer, this),
                                                     *usbSecurityModel,
                                                     serialNumber,
-                                                    purefs::dir::getUserDiskPath() / "music");
+                                                    purefs::dir::getUserDiskPath() / "storage");
```

Comment:
Same here, consider using a function for getting a full path to `storage`, instead of using raw string here.